soco/events_base.py

```diff
@@ -546,8 +546,6 @@ def unsubscribe(self):
         if self._has_been_unsubscribed or not self.is_subscribed:
             return None
 
-        self._cancel_subscription()
```

Comment:
If the speaker is now offline, `self._cancel_subscription()` will never be called with this change. which means we will not cancel auto renew attempts.

Reply:
Maybe we need a `self._cancel_subscription_success` or the similar that sets `self._timestamp` ?

Reply:
Added an optional `unconditional` parameter that takes a function in addition to `success`. It's always called after the request is complete. For now, only utilized by the unsubscription calls in both event handlers.